torch_xla/csrc/reduction.cpp

```diff
@@ -422,6 +422,11 @@ xla::XlaOp BuildAll(xla::XlaOp input, absl::Span<const xla::int64> dimensions,
   xla::XlaOp result =
       xla::Reduce(input, init_value, CreateAllComputation(shape.element_type()),
                   dimensions);
+  xla::XlaOp zeros = XlaHelpers::ScalarBroadcast<float>(
```

Comment:
For Zeros IMHO it is better to use the XLA API:

https://github.com/tensorflow/tensorflow/blob/b62ccde60d4fae90f0fa17a7b995226ada8a90e7/tensorflow/compiler/xla/client/lib/constants.h#L124